cmd/config.go

```diff
@@ -405,6 +423,21 @@ func getAddonManifests(reader *bufio.Reader) ([]string, error) {
 	return addonSlice, nil
 }
 
+func generateK8sVersionList(all bool) error {
```

Comment:
nit: maybe call it show or print list? generate makes me think something is coming back or get saved somewhere